whylog/tests/utils.py

```diff
@@ -8,3 +11,33 @@ def get_path_to_config_files(cls, prefix_path):
         rules_path = os.path.join(prefix_path, 'rules.yaml')
         log_type_path = os.path.join(prefix_path, 'log_types.yaml')
         return [parsers_path, rules_path, log_type_path]
+
+
+class TestRemovingSettings(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        #This change was caused by fails os.names() for new directories.
```

Comment:
Which change? I suppose this comment is for reviewers because it says about "change". Maybe remove it?